addons/website_forum/models/forum.py

```diff
@@ -45,11 +45,22 @@ def _get_default_faq(self):
     # description and use
     name = fields.Char('Forum Name', required=True, translate=True)
     faq = fields.Html('Guidelines', default=_get_default_faq, translate=True)
-    description = fields.Html(
+    description = fields.Text(
         'Description',
-        default='<p> This community is for professionals and enthusiasts of our products and services.'
+        default='This community is for professionals and enthusiasts of our products and services.'
                 'Share and discuss the best content and new marketing ideas,'
-                'build your professional profile and become a better marketer together.</p>')
+                'build your professional profile and become a better marketer together.')
+    introduction_message = fields.Html(
+        'Introduction Message',
+        default = """<section class="bg-info" style="height: 168px;"><div class="container">
```

Comment:
After a little chat with @tde-banana-odoo , we decided to keep the default value harcoded like this (even) it is not so beautiful. Create a new forum is a rare activity, so it can stay like that, instead of a customizable template.
Maybe on day, we will change this, but not for now. Thanks.